pkg/controller/kubelet-config/kubelet_config_controller.go

```diff
@@ -599,23 +599,23 @@ func (ctrl *Controller) syncKubeletConfig(key string) error {
 				return ctrl.syncStatusOnly(cfg, err, "could not create MachineConfig from new Ignition config: %v", err)
 			}
 			mc.ObjectMeta.UID = uuid.NewUUID()
-			_, ok := cfg.GetAnnotations()[ctrlcommon.MCNameSuffixAnnotationKey]
-			arr := strings.Split(managedKey, "-")
-			// the first managed key value 99-poolname-generated-kubelet does not have a suffix
-			// set "" as suffix annotation to the kubelet config object
-			if _, err := strconv.Atoi(arr[len(arr)-1]); err != nil && !ok {
-				if err := ctrl.addAnnotation(cfg, ctrlcommon.MCNameSuffixAnnotationKey, ""); err != nil {
-					return ctrl.syncStatusOnly(cfg, err, "could not update annotation for kubeletConfig")
-				}
+		}
```

Comment:
this diff looks weird, is it just a indentation change?

Reply:
move MCNameSuffixAnnotationKey update out of the L595, in this workarond even if the machineconfig exist, still need to update this annotation for kubeletconfig
https://github.com/openshift/machine-config-operator/blob/2092cd37b9956c52b2cdf921eec0ee88a6d5384b/pkg/controller/kubelet-config/kubelet_config_controller.go#L595